sdk/src/main/java/io/opentelemetry/sdk/trace/RecordEventsReadableSpan.java

```diff
@@ -261,22 +271,20 @@ Status getStatus() {
    */
   @VisibleForTesting
```

Comment:
Can we remove the Visible for testing and always go via SpanData even in tests?

Reply:
Let's continue this discussion at https://github.com/open-telemetry/opentelemetry-java/pull/706#discussion_r359499162